server/cookie.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
-    setSecure: function(res, name, value) {
-        res.cookie(name, value, {
+    setSecure: function(res, name, value, maxAge=undefined) {
+        const cookieProps = {
             httpOnly: true,
             secure: true,
             sameSite: 'lax'
-        });
+        };
+        if (typeof(maxAge) !== 'undefined') {
```

Comment:
I wouldn't make the maxAge default undefined. Maybe 0 to clear by default or a reasonable int to have it live by default

Reply:
The current behavior is to use session based cookies and I don't want to break that here.  But similar to the other suggestion, we should change the access_token to have an expiration time and we can address this then.